### DISCUSSION TITLE
Do you know of free databases of historic air travel?

### DISCUSSION THREAD

**Discussion by cbud** [Jxck-S/plane-notify#134] [Q&A]
Hi

Pleased this tracker stuff was done, sorry to see twitter degenerate lately.

This seems to be a real time tracker. I wonder if you can provide pointers on sources for downloading historic data for air-traffic in and out of a location?

Hopefully

Chris

**Comment by Jxck-S** [comment]
OpenSky,  and ADSBExchange have historical data. Depending on if and aircraft is blocked you can also of course to FR24 or FlightAware
